Doc/library/readline.rst

```diff
@@ -36,6 +36,15 @@ in the GNU Readline manual for information about the format and
 allowable constructs of that file, and the capabilities of the
 Readline library in general.
 
+.. note::
```

Comment:
There is another libedit-specific note in readline.rst just a few lines above this one.  Rather than adding a second note, I think it would make more sense to just have one combined one and it may make more sense to move the existing note down to this position.